src/main/java/seedu/address/storage/CustomJsonAdaptedPersonSerializer.java

```diff
@@ -11,7 +11,7 @@
  *
  * <p>The serialization process includes converting fields such as name, phone, email, role, affiliations,
  * and affiliation history to JSON fields. Additionally, it conditionally includes the shiftDays
- * field in the JSON output based on the person's role. If the role is "Doctor", the shiftDays field is
+ * field in the JSON output based on the person's role. If the role is "Staff", the shiftDays field is
```

Comment:
Just to clarify, we will not have a role "Staff" right, it is sort of an abstract class for Doctor and Nurse?

Reply:
That's right. However for the sake of testability and sanity, I have not made it abstract.

Reply:
Alright, understood, thanks!